app/models/queue_item.rb

```diff
@@ -5,17 +5,26 @@ class QueueItem < ActiveRecord::Base
   delegate :category, to: :video
   delegate :title, to: :video, prefix: :video
 
-  def rating
-    review = user.reviews.find_by(video: video)
+  validates :position, numericality: { only_integer: true }
 
-    if review
-      review.rating
-    else
-      5
-    end
+  def rating
+    user.reviews.find_by(video: video).try(:rating)
   end
 
   def category_name
     video.category.name
   end
+
+  def rating=(to_rating)
+    if rating.blank?
+      review = user.reviews.new(video: video, user: user, rating: to_rating)
+      review.skip_content_validation = true
+      review.save
+    else
+      review = Review.find_by(video: video, user: user)
+      review.skip_content_validation = true
+      review.skip_rating_validation = true
+      review.update(rating: to_rating)
```

Comment:
you can refactor this whole method by building your if/else based on an existence of review. But here when you find the review, line 25 and 26 can be bypassed by just `review.update_column(:rating, to_rating)`. There are methods that wont validate the record.